src/shaders.rs

```diff
@@ -82,7 +82,7 @@ pub fn full_shaders(
     device: &Device,
     engine: &mut WgpuEngine,
     options: &RendererOptions,
-) -> Result<FullShaders, Error> {
+) -> FullShaders {
```

Comment:
This would be a breaking change and one I'm unsure of. Where did the error go?

Reply:
There was never one. This was also mentioned in the issue, where they are multiple `Result` returning functions which just straight-up never fail

Reply:
I'm wondering why it didn't return `Infallible` before, but that's good to know.